### PR TITLE
#9 Update storage

### PR DESCRIPTION
**Pull request by li-kai**
`updateTodoListStorage` is your hook. Saving use `saveTodoListToFile` instead of `saveDataToFile`

### CODE REVIEW

File: src/main/java/seedu/todo/storage/StorageManager.java [hunk 1]
```diff
@@ -88,5 +89,39 @@ public void handleAddressBookChangedEvent(AddressBookChangedEvent event) {
             raise(new DataSavingExceptionEvent(e));
         }
     }
+    
+    // ================ AddressBook methods ==============================
```
Comment (ZhangYiJiang) [inline src/main/java/seedu/todo/storage/StorageManager.java]: You mean todo list methods?

File: src/main/java/seedu/todo/storage/TodoListStorage.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package seedu.todo.storage;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import seedu.todo.commons.exceptions.DataConversionException;
+import seedu.todo.model.ImmutableTodoList;
+
+/**
+ * Represents a storage for {@link seedu.todo.model.AddressBook}.
```
Comment (ZhangYiJiang) [inline src/main/java/seedu/todo/storage/TodoListStorage.java]: Should be TodoList

File: src/main/java/seedu/todo/storage/XmlAdaptedTask.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+import javax.xml.bind.annotation.XmlElement;
+
+import seedu.todo.commons.exceptions.IllegalValueException;
+import seedu.todo.model.tag.Tag;
+import seedu.todo.model.task.ImmutableTask;
+import seedu.todo.model.task.Task;
+
+/**
+ * JAXB-friendly version of the Person.
```
Comment (ZhangYiJiang) [inline src/main/java/seedu/todo/storage/XmlAdaptedTask.java:16]: Task, not Persob

File: src/main/java/seedu/todo/storage/XmlAdaptedTask.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+        for (Tag tag : source.getTags()) {
+            tags.add(new XmlAdaptedTag(tag));
+        }
+
+        uuid = source.getUUID();
+    }
+
+    /**
+     * Converts this jaxb-friendly adapted person object into the model's Person
```
Comment (ZhangYiJiang) [inline src/main/java/seedu/todo/storage/XmlAdaptedTask.java]: Task

File: src/main/java/seedu/todo/storage/XmlTodoListStorage.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+    private static final Logger logger = LogsCenter.getLogger(XmlTodoListStorage.class);
+
+    private String filePath;
+
+    public XmlTodoListStorage(String filePath){
+        this.filePath = filePath;
+    }
+
+	@Override
```
Comment (ZhangYiJiang) [inline src/main/java/seedu/todo/storage/XmlTodoListStorage.java]: Space not tab

**Review by ZhangYiJiang — COMMENTED** [review]
Some minor styling and copy and paste errors need to be fixed

File: src/main/java/seedu/todo/storage/XmlTodoListStorage.java (hunk 5 above)
Comment (li-kai) [reply src/main/java/seedu/todo/storage/XmlTodoListStorage.java]: God damn tabs

**Pull request merged**